etc/README.md

```diff
@@ -19,63 +19,49 @@ the [docker docs](/book/installation/docker.md#using-docker-compose).
 
 #### Adding a new metric to Grafana
 
-To set up a new metric in Reth and its Grafana dashboard:
+To set up a new metric in Reth and its Grafana dashboard (this assumes running Reth and Grafana instances):
 
 1. Add the metric to the codebase following the [metrics section](../docs/design/metrics.md#creating-metrics)
    documentation.
 
-2. Build the Reth image:
-
-    ```bash
-    docker build . -t reth:local
-    ```
-
-   Modify the [docker-compose](./docker-compose.yml) file to use your locally built image for the Reth service.
-
-3. Run Docker Compose:
-
-    ```bash
-    docker compose -f etc/docker-compose.yml -f etc/lighthouse.yml up -d
-    ```
-
-4. Access Grafana:
+2. Access Grafana:
```

Comment:
just use `1.` on all of these so we dont need to update them all the time. markdown will automatically convert them on render